src/pages/home/report/EmojiPickerMenu/index.js

```diff
@@ -136,6 +136,9 @@ class EmojiPickerMenu extends Component {
                     renderItem={this.renderItem}
                     keyExtractor={item => `emoji_picker_${item.code}`}
                     numColumns={this.numColumns}
+                    removeClippedSubviews
+                    maxToRenderPerBatch={this.numColumns}
+                    windowSize={3}
```

Comment:
What does this have to do with your issue? This just seems like a FlatList optimization that doesn't have anything do with your changes. NAB but I think it's better to do this in a separate PR.

Also, can you clarify why you're using 3 for windowSize? Not good to have hard coded numbers so good to clarify with a comment.

Reply:
This component is very laggy. Author of the component mentioned this so he disabled the animations. I put these changes to make the animation smooth so that I can enable animation on this component. 

Window size will not be reused so I put it directly over here. It just means to create three Windows where one window is equal to visible area of component.

Reply:
I forgot to mention that there are other PRs on emoji picker which is optimizing it So I don't need to submit a PR for now. If in future issues persists with Emoji Picker I will handle them as a separate issue.

Reply:
+1 for @jasperhuangg comments here. 

@parasharrajat appreciate you adding value here but please do limit changes to ones affecting the issue you are assigned so that we can manage which bugs and issues are being handled and where. random fixes make it much harder to track when a bug was fixed.

Reply:
I added these changes as per request https://github.com/Expensify/Expensify.cash/issues/2335#issuecomment-824089744

Reply:
Ah ok, thanks I missed this comment. No need to remove changes but if we are increasing the scope we need to add that context to the description of the original issue so we should:

1. Update the issue you are working on to show each problem we are solving (show that it is broken)
2. Ask QA to test the same flows and prove that it is fixed.

Otherwise, we're just adding random stuff and it's hard to have a good history.

Reply:
Sure, Thanks. Updating now.